doc/examples/animations/lib/main.dart

```diff
@@ -13,7 +14,11 @@ void main() async {
 
   final Vector2 size = await Flame.util.initialDimensions();
   final game = MyGame(size);
-  runApp(game.widget);
+  runApp(
+    GameWidget(
```

Comment:
idea: how about we provide a runGame helper function that just calls runApp with an empty GameWidget? that would make most of our examples a bit simpler. of course if you want to customize anything on the widget then you do runApp with the game widget

Reply:
Nice idea, I wouldn't add this here since this can scale for more stuff on the bootstrap of the app.